clusterloader2/testing/load/config.yaml

```diff
@@ -1,36 +1,40 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$BIG_GROUP_SIZE := 250}}
 {{$MEDIUM_GROUP_SIZE := 30}}
 {{$SMALL_GROUP_SIZE := 5}}
-{{$PODS_PER_NODE := 30}}
 {{$NODES_PER_NAMESPACE := 100}}
-#Variables
+{{$PODS_PER_NODE := 30}}
+{{$TEST_THROUGHPUT := 10}}
 # bigRCs - 1/4 of all pods should be in big RCs.
-{{$bigRCs := MultiplyInt .Nodes 0.03}}
+{{$BIG_RCS_PER_NAMESPACE := 3}}
 # mediumRCs - 1/4 of all pods should be in medium RCs.
-{{$mediumRCs := MultiplyInt .Nodes 0.25}}
+{{$MEDIUM_RCS_PER_NAMESPACE := 25}}
 # smallRCs - 1/2 of all pods should be in small RCs.
-{{$smallRCs := MultiplyInt .Nodes 3}}
-{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
-{{$bigRCsPerNamespace := DivideInt $bigRCs $namespaces}}
-{{$mediumRCsPerNamespace := DivideInt $mediumRCs $namespaces}}
-{{$smallRCsPerNamespace := DivideInt $smallRCs $namespaces}}
-{{$totalPodsRounded := AddInt (MultiplyInt $bigRCs $BIG_GROUP_SIZE) (MultiplyInt $mediumRCs $MEDIUM_GROUP_SIZE) (MultiplyInt $smallRCs $SMALL_GROUP_SIZE)}}
-# saturationQPS - qps for RC operations should be an equivalent to 10 [pods/s].
-{{$saturationQPS := DivideFloat (AddInt $bigRCs $mediumRCs $smallRCs) (DivideFloat $totalPodsRounded 10)}}
-#Test
+{{$SMALL_RCS_PER_NAMESPACE := 300}}
+
+#Variables
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+{{$totalPods := MultiplyInt $namespaces $NODES_PER_NAMESPACE $PODS_PER_NODE}}
+# saturationTime - time for RC saturation should be an equivalent to 10 [pods/s] qps.
```

Comment:
This comment doesn't provide any value imho

Reply:
Comment removed.